Source/Core/DolphinNoGUI/MainNoGUI.cpp

```diff
@@ -172,6 +173,18 @@ class PlatformX11 : public Platform
     Atom wmProtocols[1];
     wmProtocols[0] = XInternAtom(dpy, "WM_DELETE_WINDOW", True);
     XSetWMProtocols(dpy, win, wmProtocols, 1);
+    pid_t pid = getpid();
+    XChangeProperty(dpy, win, XInternAtom(dpy, "_NET_WM_PID", False), XA_CARDINAL, 32,
+                    PropModeReplace, (unsigned char*)&pid, 1);
+    char hostname[HOST_NAME_MAX] = "";
+    if (!gethostname(hostname, sizeof(hostname)))
+    {
+      XTextProperty wmClientMachine = {.value = (unsigned char*)hostname,
```

Comment:
```cpp
reinterpret_cast
```

variables should follow `snake_case`.

C++17 also doesn't support designated initializers for structs (C++20 may add this to the language, however)